Controller/CRUDController.php

```diff
@@ -1349,6 +1348,28 @@ protected function preList(Request $request)
     }
 
     /**
+     * Sets the admin form theme to form view. Used for compatibility between Symfony versions.
+     *
+     * @param FormView $formView
+     */
+    protected function setFormTheme(FormView $formView)
```

Comment:
That's true, but we will remove the feature to set the form rendering in the controller. This will be done in the template files with the next major.